### PR TITLE
feat: support for tweakpane v4

### PR DESCRIPTION
**Pull request by kitschpatrol**
Thanks for this plugin.

I ported it to Tweakpane 4.x. This is a breaking change for users of Tweakpane 3.x.

The updates were mostly based on the following documentation:

- https://cocopon.github.io/tweakpane/migration/v4/
- https://github.com/cocopon/tweakpane/issues/396#issuecomment-1418929095
- https://github.com/tweakpane/plugin-essentials/commit/be4394e25c9fe6744627f3a1e0b54f215b0fee60

I made some minor structural changes to more closely reflect the tweakpane [plugin-template](https://github.com/tweakpane/plugin-template).

### CODE REVIEW

**Comment by 0b5vr** [comment]
Thank you for the PR! will look into it

**Comment by 0b5vr** [comment]
I wondered why eslint has to be bumped but

> I made some minor structural changes to more closely reflect the tweakpane plugin-template.

It's very nice of you. 🥰

File: package.json [hunk 1]
```diff
@@ -1,31 +1,26 @@
 {
   "name": "@0b5vr/tweakpane-plugin-rotation",
-  "version": "0.1.1",
+  "version": "1.0.0",
```
Comment (0b5vr) [inline package.json]:
```suggestion
  "version": "0.2.0",
```

Minor bump would be preferred. Zero versioning.

File: src/RotationInputPluginQuaternion.ts [hunk 2]
```diff
@@ -88,6 +86,6 @@ RotationInputPluginQuaternionParams
       pickerLayout: picker ?? 'popup',
       value,
       viewProps: viewProps,
-    } );
+    } )  as unknown as ValueController<Quaternion>; // TODO;
```
Comment (0b5vr) [inline src/RotationInputPluginQuaternion.ts:89]: nice TODO. I couldn't come up with a viable solution.

**Review by 0b5vr — APPROVED** [review]
Looks PERFECT. Thanks a lot!!

**Comment by kitschpatrol** [comment]
Thanks for sharing your work!

**Pull request merged**